package/kedro_viz/models/flowchart.py

```diff
@@ -427,7 +427,7 @@ def __post_init__(self, task_node: TaskNode):
         # if a node doesn't have a user-supplied `_name` attribute,
         # a human-readable run command `kedro run --to-nodes/nodes` is not available
         if kedro_node._name is not None:
-            self.run_command = f'kedro run --to-nodes="{kedro_node._name}"'
```

Comment:
Why is it important to remove these quotes?

Reply:
Ahdra mentioned as per the new cli commands we don't require quotes.